src/whylogs/viewer/index-hbs-cdn-all-in-jupyter-full-summary-statistics.html

```diff
@@ -0,0 +1,328 @@
+            best experienced on a desktop computer. Please try maximizing this window or switching to another device. We
+            are working on adding support for a larger variety of devices.
+          </p>
+        </div>
+      </div>
+    {{{{/raw}}}}
+  </script>
+
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
+
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/6.7.0/d3.min.js" integrity="sha512-cd6CHE+XWDQ33ElJqsi0MdNte3S+bQY819f7p3NUHgwQQLXSKjE4cPZTeGNI+vaxZynk1wVU3hoHmow3m089wA==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+
+  <script>
+    function registerHandlebarHelperFunctions() {
+      //helper fun
+      function formatLabelDate(timestamp) {
+        const date = new Date(timestamp);
+        const format = d3.timeFormat("%Y-%m-%d %I:%M:%S %p %Z");
+        return format(date);
+      }
+
+      function abbreviateNumber(value) {
+        let newValue = value,
+        suffixe = ""
+        if (value >= 1000) {
+            let suffixes = ["", "K", "M", "B","T"];
+            let suffixNum = Math.floor( (""+value).length/3 );
+            let shortValue = '';
+            for (let precision = 2; precision >= 1; precision--) {
+                shortValue = parseFloat( (suffixNum != 0 ? (value / Math.pow(1000,suffixNum) ) : value).toPrecision(precision));
+                let dotLessShortValue = (shortValue + '').replace(/[^a-zA-Z 0-9]+/g,'');
+                if (dotLessShortValue.length <= 2) { break; }
+            }
+            if (shortValue % 1 != 0)  shortValue = shortValue.toFixed(1);
+            newValue = shortValue;
+            suffixe = suffixes[suffixNum]
+        }
+        return {value, newValue, suffixe};
+      }
+
+      function formatBytes(bytes, decimals = 2) {
+        let newValue,
+        suffixe = ""
+        if (bytes === 0) return '0 Bytes';
+
+        const k = 1024;
+        const dm = decimals < 0 ? 0 : decimals;
+        const sizes = ['Bytes', 'KiB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+
+        const i = Math.floor(Math.log(bytes) / Math.log(k));
+        newValue = parseFloat((bytes / Math.pow(k, i)).toFixed(dm));
+        suffixe = sizes[i]
+        return {bytes, newValue, suffixe};
+      }
+
+      const numberWithSuffixe = (number, newNumber, suffixe) =>
+        `<div class="tooltip-full-number">
+           <div class="statistic-number">${newNumber}
+             <div class="statistic-measurement">${suffixe}</div>
+           </div>
+           <span class="tooltiptext">${number}</span>
+         </div>`
+
+      const randomNumbers = (range) => Math.floor(Math.random() * range)
+
+      Handlebars.registerHelper("observations", function () {
+        const {value, newValue, suffixe} = abbreviateNumber(randomNumbers(900000000))
```

Comment:
These are still random numbers as placeholders? Let me know when the reports are using the profile data.

Reply:
fixed